MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/CustomizeUserLocationActivity.java

```diff
@@ -113,22 +124,63 @@ private void enableGps() {
   }
 
   @Override
-  public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
-    permissionsManager.onRequestPermissionsResult(requestCode, permissions, grantResults);
+  public void onRequestPermissionsResult(
+    int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+    if (requestCode == PERMISSIONS_LOCATION) {
```

Comment:
Forgot to switch back to using the PermissionsManager amongst my experimenting. Latest commit uses it.